src/Features/Core/Portable/PullMemberUp/MembersPuller.cs

```diff
@@ -275,6 +278,21 @@ private static ISymbol GetSymbolsToPullUp(MemberAnalysisResult analysisResult)
                         solution.GetDocumentId(syntax.SyntaxTree),
                         cancellationToken).ConfigureAwait(false);
 
+                    // add import symbol annotations for all the code we may be moving
+                    newDestination = newDestination.WithAdditionalAnnotations(
+                        semanticModel.GetAllDeclaredSymbols(syntax, cancellationToken)
+                        .Select((symbol) =>
+                        {
+                            // prefer the convert because it can handle param and return types
+                            // but use memberType in case of fields/variable declaration.
+                            var type = symbol.ConvertToType(compilation);
+                            if (type.Equals(compilation.ObjectType))
+                            {
+                                type = symbol.GetMemberType();
```

Comment:
It is a commit for all the tests.
Could you write tests that imports some other namespaces? (Not 'System', but like 'System.Linq, or System.Threading')

This is because System is so widely used, at this line, if the symbol is a method, then it would return the 'System.Func',
which cause 'System' is imported later.
However, because all your tests now are importing 'System', if the member is
`void Bar() { var c = Uri.TryCreate() ... }`
'System' could be imported by 'Func', not 'Uri'



I saw below you are calling
semanticModel.GetAllDeclaratedSymbol()

But please

Reply:
Also, semanticModel.GetAllDeclaredSymbols will only returns the 'declared' symbol,
If I have a method like
`void bar() { Uri.TryCreate(...) }`
I really worry 'Uri' won't be passed in this method